website/docs/r/environment.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: "gitlab"
+page_title: "Gitlab: gitlab_environment_file"
+sidebar_current: "docs-gitlab-environment-file"
+---
+
+# gitlab\_environment
+
+This resource allows you to create and manage Gitlab environments.
+
+## Example Usage
+
+```hcl
+resource "gitlab_group" "this" {
```

Comment:
You can remove the group from the examples for both resources.

Reply:
No, this ensures that the example is working.